packages/forma-36-website/src/content/guidelines/contributing.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: 'Contributing to Forma 36'
+slug: '/guidelines/contributing/'
+---
+
+Forma 36 is an open-source design system by Contentful and we welcome contributions from our community.
+If you have an idea that could help other users in this page we will teach you how to do it.
+
+## Table of contents
+
+- [Before contributing](#before-contributing)
+  - [Check if your idea is already present](#check-if-your-idea-is-already-present)
+  - [Could an existing component be extended to realize your idea?](#could-an-existing-component-be-extended-to-realize-your-idea)
+- [Contribution Process](#contribution-process)
+  - [Step 1: Open a Github issue](#step-1-open-a-github-issue)
+  - [Step 2: Proposal Review](#step-2-proposal-review)
+  - [Step 3: Open a Pull Request](#step-3-open-a-pull-request)
+  - [Step 4: Code & Design Review](#step-4-code--design-review)
+  - [Step 5: Merge the changes to the main branch](#step-5-merge-the-changes-to-the-main-branch)
+- [Bugs & Light contributions](#bugs--light-contributions)
+  - [Examples of light contributions](#examples-of-light-contributions)
+
+## Before contributing
+
+### Check if your idea is already present
+
+Before writing a proposal or any code, we ask you to go around this website or our Github repo and
+check if your idea is not yet implemented.
+It’s also a good idea to take a look on the current open proposals,
+maybe there is something similar to your idea there and you could work together with that proposer.
+
+### Could an existing component be extended to realize your idea?
+
+During your research, did you find a component that almost does what you have in mind?
+If yes, consider proposing an update or refactor of this component.
+
+## Contribution Process
+
+Now that you checked the library and know if you want to create a new component
+or extend an existing one it’s time to go through the contribution process.
+
+### Step 1: Open a Github issue
+
+The first step is to [open a new issue in Github](https://github.com/contentful/forma-36/issues/new/choose) and choose the correct template.
+Remember to describe your idea in detail and maybe use images to support it.
+
+### Step 2: Proposal Review
+
+Forma 36’s internal team checks the issues in the repo regularly and we will provide you with feedback to all proposals.
+After some discussion we will approve the proposal and you can work on it. We will help you in anything you need.
+
+It’s a good idea to look into the other components’ code
+and our [Code Style Guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md) to write your code following our convetions.
+
+### Step 3: Open a Pull Request
```

Comment:
Maybe it's worth adding info around `feat` vs `fix` and how it would be reflected in the lib. so `minor` vs `fix`? 
Dunno, maybe also add that major, breaking changes are closed for external contributions? (I guess they are?)

Reply:
I prefer to keep it short
The versioning method we seems to be a detail that the contributors don't need to know
or if they want to know they can ask us

I just don't want this to be very long and because it's long people will avoid it